core/src/main/resources/reference.conf

```diff
@@ -193,6 +193,9 @@ cassandra-journal {
     # scanning during recovery of persistent actor.
     scanning-flush-interval = 30s
 
+    # The actor responsible for writing a tag is stopped if the tag isn't used for this duration.
+    stop-tag-writer-when-idle = 5s
```

Comment:
I don't have a strong reason for using 5s, but I can't see that there is much overhead of stopping it. It will be kept alive as long as there is a PersistentActor alive that used the tag.